ably/auth_test.go

```diff
@@ -10,11 +10,11 @@ import (
 var _ = Describe("Auth", func() {
 	Describe("RequestToken", func() {
 		It("gets a token from the API", func() {
-			req := client.Auth.CreateTokenRequest()
+			req := client.Auth().CreateTokenRequest()
```

Comment:
I'm interested to see that Auth is not a property / attribute, but instead a function.  Why is that? That is inconsistent with other client libraries at present because `Auth` is automatically created for every `Client`.